ci/variables.md

```diff
@@ -0,0 +1,33 @@
+# Variables
+
+A list of variables and their descriptions used by these pipelines
+
+## Secrets
+
+These variables are secrets, provided by the hosted Concourse, or ones we have set up
+
+### Github Access Token
+
+Stored under: `github-access-token`
+
+An access token for cloning public + private repos, and sending `status` updates to PRs.
+
+This is associated with the `govwifi-jenkins` Github user account.
```

Comment:
Is there a task to create a github-concourse user account?

Reply:
nope. does one need making? The current user account could be renamed to `govwifi-ci`